.travis.yml

```diff
@@ -8,5 +7,10 @@ node_js:
     - 10
     - 11
     - 12
+
+script:
+    - if [ $TRAVIS_NODE_VERSION -ge 8 ]; then node Makefile.js lint; fi
+    - node Makefile.js test
```

Comment:
out of curiosity: this was added since eslint v6 don't support node <8?

we may consider dropping node <8, as it has been EOL.

Reply:
@aladdin-add I assume that was the case (but hopefully @mysticatea can confirm). I think it makes sense to release a minor version of espree with this functionality if we can, and then we could do a major release to drop Node <8.